### PR TITLE
Added support for FIQL + small code revisions

### PR DESCRIPTION
**Pull request by thijskok**

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -59,7 +59,7 @@ public function __construct($instance, $username, $password)
     protected function client()
     {
         return $this->client ?? new \GuzzleHttp\Client([
-            'base_uri' => $this->instance . '/',
```
Comment (stefanius) [inline src/Client.php:62]: Wasn't this mandatory?

File: tests/FIQLTest.php [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+        // When
+        $builder = (new FIQL);
+
+        // Then
+        $this->assertInstanceOf(FIQL::class, $builder);
+    }
+
+    /** @test */
+    public function it_should_generate_a_wiql_query()
```
Comment (stefanius) [inline tests/FIQLTest.php:25]: wiql or fiql?

File: tests/FIQLTest.php [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+use PHPUnit\Framework\TestCase;
+use TestMonitor\TOPdesk\Builders\FIQL\FIQL;
+use TestMonitor\TOPdesk\Builders\FIQL\Field;
+use TestMonitor\TOPdesk\Builders\FIQL\Operator;
+
+class FIQLTest extends TestCase
+{
+    /** @test */
+    public function it_should_return_a_fiql_builder()
```
Comment (stefanius) [inline tests/FIQLTest.php:13]: wiql or fiql?

File: tests/FIQLTest.php (hunk 2 above)
Comment (stefanius) [inline tests/FIQLTest.php]: wiql or fiql?

File: src/Client.php (hunk 1 above)
Comment (thijskok) [reply src/Client.php:62]: Nope, works fine. In fact, this was adding an additional slash.

File: tests/FIQLTest.php (hunk 2 above)
Comment (thijskok) [reply tests/FIQLTest.php]: Fixed!

**Review by stefanius — APPROVED** [review]

**Pull request merged**